olive/passes/onnx/gptq_utils.py

```diff
@@ -0,0 +1,308 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+import logging
+import math
+import os
+
+import torch
+import torch.nn as nn
+from torch.onnx.symbolic_helper import _get_tensor_dim_size, _get_tensor_sizes
```

Comment:
are there any public function? if it is private, the signature or name might be changed from different version.

Reply:
These func is used to remove `no shape inference` error when export, I didn't find other function, is there any better way to add shape info?

Reply:
could we just copy these private function here? I am afraid the private function signature changes would break Olive in future.

Reply:
I see, I will check here, or just delete these funcs, cause they just omit some ugly export warning.